lib/graphql/schema.rb

```diff
@@ -1631,6 +1631,14 @@ def multiplex_analyzers
         find_inherited_value(:multiplex_analyzers, EMPTY_ARRAY) + own_multiplex_analyzers
       end
 
+      def sanitized_printer(new_sanitized_printer = nil)
+        if new_sanitized_printer
+          @own_sanitized_printer = new_sanitized_printer
+        else
+          @own_sanitized_printer || GraphQL::Language::SanitizedPrinter
```

Comment:
At some point, we may get bit because this isn't inherited. If that happens, we can use `find_inherited_value` as some of the other methods do. But it's ok with me to keep it this way for now.